### PR TITLE
Wifi

### PR DESCRIPTION
**Pull request by jorgesg82**
Added the capability to connect to a wifi.

CMakeLists.txt must be modified to be able to grab SSID and password from cmake command, using `-DWIFI_SSID` and `-DWIFI_PASSWORD`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,13 +5,14 @@ Firmware designed for the pico of the IDC project
 
 # Building:
   
-1. create and enter desired build directory
-2. run `cmake ..`
-3. run `make`
+1. create your build directory `mkdir build`
+2. run `cmake -DPICO_BOARD=pico_w  -DWIFI_SSID='SSID' -DWIFI_PASSWORD='PASS' ..`
```
Comment (AmancioG) [inline README.md:9]: This is not necessary. all those flags are already inside of the CMakeList file

**Review by AmancioG — CHANGES_REQUESTED** [review]
I would change the readme, everything else looks fine

File: README.md (hunk 1 above)
Comment (jorgesg82) [reply README.md:9]: pico_w is inside CMakeLists, but SSID and PASSWORD isn't. In CMakeLists is specified to GRAB those secrets and use it on code

Comment (AmancioG) [reply README.md:9]: You are correct, sorry. Nevertheless, I would include the lines

set(WIFI_SSID SSID)
set(WIFI_PASSWORD PASS)

into the CMakeList.txt file, for a cleaner cmake command

Comment (jorgesg82) [reply README.md:9]: I would not, due to the increased configuration complexity for the end user

**Review by AmancioG — APPROVED** [review]

**Pull request merged**